Resources/public/js/views/dashboard/ez-dashboardblockallcontentview.js

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-dashboardblockallcontentview', function (Y) {
+    'use strict';
+
+    /**
+     * Provides the All Content Dashboard Block View class
+     *
+     * @module ez-dashboardblockallcontentview
+     */
+    Y.namespace('eZ');
+
+    var BLOCK_IDENTIFIER = 'all-content',
+        CLASS_LOADING = 'is-loading',
+        CLASS_ROW_SELECTED = 'is-row-selected',
+        SELECTOR_ROW = '.ez-allcontent-block-row',
+        SELECTOR_CONTENT = '.ez-allcontent-block-content',
+        EVENTS = {};
+
+    EVENTS[SELECTOR_ROW] = {tap: '_uiShowRowOptions'};
+
+    /**
+     * The all content dashboard block view
+     *
+     * @namespace eZ
+     * @class DashboardBlockAllContentView
+     * @constructor
+     * @extends eZ.DashboardBlockBaseView
+     */
+    Y.eZ.DashboardBlockAllContentView = Y.Base.create('dashboardBlockAllContentView', Y.eZ.DashboardBlockBaseView, [Y.eZ.AsynchronousView], {
+        events: EVENTS,
```

Comment:
as far as I can remember, when using the asynchronous view extension, you can not define the dom event handlers this way otherwise the ones defined in asynchronous view (for the retry button) are completely overriden. You have to write it like https://github.com/ezsystems/PlatformUIBundle/blob/206d055a33cc735cfd050a90bf708113d5f8ed77/Resources/public/js/views/tabs/ez-locationviewlocationstabview.js#L39